tools/project_manager/diff.php

```diff
@@ -61,6 +62,19 @@
     $R_label = $R_round->id;
 }
 
+$L_format = is_format_round($L_label);
+$R_format = is_format_round($R_label);
+$format = array_get($_GET, "format", "default");
+if($format == "keep")
```

Comment:
Please use brackets in all if/then/else statements if you're going to use them in any of them.